lib/client.js

```diff
@@ -40,19 +41,37 @@ function* createClient(rabbitmqUrl, options) {
     consume,
     listen,
     publish,
-    close: co.wrap(function* close() {
+    close: co.wrap(function* close(forceClose = true) {
```

Comment:
Could you update the README also ?
With a big warning indicating that, by default, this method will exit the process.
If this method is used in unit tests, mocha will stop silently on the first bus.close()

Reply:
After talking about this with others, we've actually decided to define `forceClose` to `false` by default, as we only need it in the `close` event handler.